extension.js

```diff
@@ -294,3 +294,135 @@ const getParamsFromGithubRaw = (url) => {
     const installedExtensions = getLocalStorageDataFromKey(LOCALSTORAGE_KEYS.installedExtensions, []);
     installedExtensions.forEach((extensionKey) => initializeExtension(extensionKey));
 })();
+const ITEMS_PER_REQUEST = 100;
+async function storeThemes() {
+    const BLACKLIST = await Blacklist();
```

Comment:
In `storeExtensions` and `storeThemes`, it calls the `Blacklist()` twice. We could likely just call it once somewhere else and have it cached.

Reply:
@theRealPadster Done (made a call in initalizePreload to add blacklist to session storage)